docs/api.md

```diff
@@ -64,6 +64,7 @@
     + [page.setUserAgent(userAgent)](#pagesetuseragentuseragent)
     + [page.setViewport(viewport)](#pagesetviewportviewport)
     + [page.title()](#pagetitle)
+    + [page.touchScreen](#pagetouchscreen)
```

Comment:
the page.touchScreen feels slightly weird. Paul, any naming ideas? @paulirish

Reply:
we can do `page.touchscreen`, all lowercase because apparently [Touchscreen](https://en.wikipedia.org/wiki/Touchscreen) is one word